include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_scan.h

```diff
@@ -214,12 +207,12 @@ single_pass_scan_impl(sycl::queue __queue, _InRange&& __in_rng, _OutRange&& __ou
 }
 
 // The generic structure for configuring a kernel
-template <std::uint16_t DataPerWorkItem, std::uint16_t WorkGroupSize, typename KernelName>
+template <std::uint16_t _WorkGroupSize, std::uint16_t _ElemsPerWorkItem, typename _KernelName>
```

Comment:
Here's the upstream definition of `kernel_param`:
```c++
namespace oneapi::dpl::experimental::kt
{

template <std::uint16_t __data_per_work_item, std::uint16_t __work_group_size,
          typename _KernelName = oneapi::dpl::execution::DefaultKernelName>
struct kernel_param
{
    static constexpr std::uint16_t data_per_workitem = __data_per_work_item;
    static constexpr std::uint16_t workgroup_size = __work_group_size;
    using kernel_name = _KernelName;
};

} // namespace oneapi::dpl::experimental::kt
```

So I think the order should be reverted & the test updated.